docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -65,18 +124,17 @@ include::{sourcedir}/clustered/client/src/test/java/org/ehcache/clustered/client
 <2> Returns the `ClusteringServiceConfigurationBuilder` instance.
 <3> `defaultServerResource(String)` on ClusteringServiceConfigurationBuilder instance sets the default server resource for the cache manager.
     From the example, cache manager sets its default server resource to server storage resource identifier *primary-server-resource* in the server.
-    The server storage resource identifier in turn will be used to create EhCache resource pools.
-<4> Adds a resource pool for the cache manager with the specified name(*resource-pool-a*) and size(*128B*) consumed out of a named server storage resource(*secondary-server-resource*)
-    already existing on the server.
-<5> Adds a resource pool for the cache manager with the specified name(*resource-pool-b*) and size(*128B*).
-    Since the server storage identifier is not explicitly passed, this resource pool will be consumed out of default server resource provided in Step 3.
-    This demonstrates that a cache manager with clustering support can have multiple resource pools created out of several server storage resources.
+    The server storage resource identifiers will be used to create resource pools.
+<4> Adds a resource pool for the cache manager with the specified name(*resource-pool-a*) and size(*28MB*) consumed out of the  named server resource *secondary-server-resource*
+<5> Adds a resource pool for the cache manager with the specified name(*resource-pool-b*) and size(*32MB*).
```

Comment:
These descriptions are too long for callouts. There should be more upfront definition and explanation and the callouts only serve as illustration.